src/main.rs

```diff
@@ -1,35 +1,52 @@
+mod generator;
 mod geo;
 mod mission;
 mod parser;
+mod print;
 mod robot;
 
-use std::fmt;
 use std::io;
 
-use geo::orientation::Orientation::{self, East, North, South, West};
+use clap::{App, Arg, SubCommand};
+
+use generator::Generator;
 use mission::Mission;
 use parser::MissionPlan;
-use robot::Robot;
-
-impl std::fmt::Display for Orientation {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        let text = match self {
-            North => 'N',
-            East => 'E',
-            South => 'S',
-            West => 'W',
+
+const SEED: u64 = 12345;
```

Comment:
why a constant?

Reply:
So that it’s repeatable. I should probably expose it as a CLI flag, so various different runs are possible.

Reply:
Done that.